climpred_skeleton/core.py

```diff
@@ -28,6 +35,57 @@ def __init__(
         self._all_verifs = self._observation['time'].data
         self._all_inits = self._initialized['init'].data
 
+    def _convert_to_cftime_index(self, calendar: str = 'DatetimeProlepticGregorian'):
```

Comment:
Two questions here:

(1) Is it bad form here to overwrite `self._initialized` and `self._observation`? Typically it's good to return objects with methods, but this is done completely internally during initialization.

(2) Is it okay to have the time conversion under `Verification`? This definitely breaks the Single-responsibility principle (https://en.wikipedia.org/wiki/SOLID) but I don't want to split out a separate Time class. I think that's too much. Maybe we have a TimeManagement class once we expand out to dealing with differing calendars, handling `time_bounds`, etc.